### ISSUE TITLE
Oddities Using F12 (Go To Defintion) With Value Tuples

### ISSUE THREAD

**Issue by JasonBock** [dotnet/roslyn#21805]
**Version Used**: 
VS 2017, 15.3.2

**Steps to Reproduce**:
1) Write this code:
```
var x = (1, 2);
```
2) Move the caret next to the tuple, and hit F12.

**Expected Behavior**:
I should see a `ValueTuple<T1, T2>` metadata definition in a new file.

**Actual Behavior**:
I get the error: "Cannot navigate to the symbol under the caret."

![image](https://user-images.githubusercontent.com/904213/29853987-e1e09ff6-8d08-11e7-95e8-1338c2f8730e.png)

Note that if I do this:
```
var x = 2;
```

And move the caret to the 2 and hit F12, that works as expected - I get a "Int32 [from metadata]" file tab with `Int32` in the file.

Side issue - if I write this code:
```
var x = new ValueTuple<int, int>(1, 2);
```
And press F12 on `ValueTuple`, I get a metadata file, but the type name isn't shown:
![image](https://user-images.githubusercontent.com/904213/29854063-4998698a-8d09-11e7-8961-4eba16e62a64.png)

**Comment by jcouv** [comment]
Relates to https://github.com/dotnet/roslyn/issues/14524

**Comment by jcouv** [comment]
Aleksey is working on a fundamental change to the tuple symbol model (to have tuple types be considered generic), which may affect this. Let's wait for that change before investigating further.

**Comment by gafter** [comment]
Yes, this would likely be fixed as a side-effect of the changes proposed for #20648

**Comment by CyrusNajmabadi** [comment]
I properly see this source for ValueTuple:

<img width="809" height="173" alt="Image" src="https://github.com/user-attachments/assets/18dc8db3-7814-496d-9881-8f674ec130a1" />
